charts/hono/README.md

```diff
@@ -51,6 +51,8 @@ kubectl create namespace hono
 The chart can then be installed to name space `hono` using release name `eclipse-hono`:
 
 ```bash
+helm repo add eclipse-iot https://eclipse.org/packages/charts
+helm repo update
```

Comment:
FMPOV this is already covered by the *Prerequisites* section referring to the website. If we repeat these instructions here we are running at risk of inconsistencies if we change e.g. the repo URL or something. I would therefore rather omit it here ...